CMakeLists.txt

```diff
@@ -70,6 +70,7 @@ set(BOOST_FILESYSTEM_SOURCES
 )
 
 at_boost_add_library(at_boost_filesystem ${BOOST_FILESYSTEM_SOURCES})
+target_compile_definitions(at_boost_filesystem PRIVATE BOOST_FILESYSTEM_NO_CXX20_ATOMIC_REF)
```

Comment:
since boost enables it by default disables cxx20_atomic_ref. details here https://airtime.atlassian.net/wiki/spaces/ENG/pages/3110993940/Using+Boost+Cmake+Issues#.-Reason-to--use-boost-cmake